frontend/src/component/demo/demo-setup.ts

```diff
@@ -67,75 +68,48 @@ export const gradualRollout = async () => {
-export const variants = async () => {
-    const featureId = 'demoApp.step4';
-
-    await ensureUserIdContextExists();
+const deleteStrategy = (featureId: string, strategyId: string) =>
+    fetch(
+        formatApiPath(
+            `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/strategies/${strategyId}`
+        ),
+        { method: 'DELETE' }
+    );
 
-    const { variants }: IFeatureToggle = await fetch(
+const deleteOldStrategies = async (featureId: string) => {
+    const results = await fetch(
         formatApiPath(
-            `api/admin/projects/${PROJECT}/features/${featureId}?variantEnvironments=true`
+            `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/strategies`
         )
     ).then(res => res.json());
 
-    if (!variants.length) {
-        await fetch(
-            formatApiPath(
-                `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/variants`
-            ),
-            {
-                method: 'PATCH',
-                headers: {
-                    'Content-Type': 'application/json',
-                },
-                body: JSON.stringify([
-                    {
-                        op: 'add',
-                        path: '/0',
-                        value: {
-                            name: 'red',
-                            weightType: 'variable',
-                            weight: 333,
-                            overrides: [],
-                            stickiness: 'default',
-                            payload: {
-                                type: 'string',
-                                value: 'red',
-                            },
-                        },
-                    },
-                    {
-                        op: 'add',
-                        path: '/1',
-                        value: {
-                            name: 'green',
-                            weightType: 'variable',
-                            weight: 333,
-                            overrides: [],
-                            stickiness: 'default',
-                            payload: {
-                                type: 'string',
-                                value: 'green',
-                            },
-                        },
-                    },
-                    {
-                        op: 'add',
-                        path: '/2',
-                        value: {
-                            name: 'blue',
-                            weightType: 'variable',
-                            weight: 333,
-                            overrides: [],
-                            stickiness: 'default',
-                            payload: {
-                                type: 'string',
-                                value: 'blue',
-                            },
-                        },
-                    },
-                ]),
-            }
```

Comment:
This means we no longer have one of red, green, blue on the demo app on a clean slate, right? Will it default to black? I honestly don't remember.

I'm OK with it, but it's something we should be aware of and maybe align with @EliseBrevald and @nicolaesocaciu

Reply:
I didn't know the context of those so glad your brought it up. I can bring those back as fallback. It shouldn't change much for the strategy variants.